src/features/quiz/answers/answerSlice.ts

```diff
@@ -0,0 +1,45 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { RootState } from "../../../app/store";
+import { Answer } from "./../../../app/types";
+
+export const mockAnswer: Answer[] = [
+  {
+    promptId: 0,
+    answerId: 2,
+  },
+];
+
+export interface AnswerState {
+  currentAnswer: number | undefined;
+  answers: number[];
+}
+
+const initialState: AnswerState = {
+  currentAnswer: undefined,
```

Comment:
Should we rename the state into "correctAnswer"?

Reply:
joo